crates/nu-std/lib/help.nu

```diff
@@ -494,190 +500,195 @@ export def "help operators" [
-        for example in $command.examples {
-            print ""
-            print $"  ($example.description)"
-            print $"  > ($example.example | nu-highlight)"
-            if not ($example.result | is-empty) {
-                for line in (
-                    $example.result | table | if ($example.result | describe) == "binary" { str join } else { lines }
-                ) {
-                    print $"  ($line)"
-                }
-            }
+        print "This command:"
+        []
+        if ($command.creates_scope) {
+            print $"- (ansi cyan)does create(ansi reset) a scope."
+        } else {
+            print $"- (ansi cyan)does not create(ansi reset) a scope."
+        }
+        if ($command.is_builtin) {
+            print $"- (ansi cyan)is(ansi reset) a built-in command."
+        } else {
+            print $"- (ansi cyan)is not(ansi reset) a built-in command."
+        }
+        if ($command.is_sub) {
+            print $"- (ansi cyan)is(ansi reset) a subcommand."
+        } else {
+            print $"- (ansi cyan)is not(ansi reset) a subcommand."
+        }
+        if ($command.is_plugin) {
+            print $"- (ansi cyan)is part(ansi reset) of a plugin."
+        } else {
+            print $"- (ansi cyan)is not part(ansi reset) of a plugin."
+        }
+        if ($command.is_custom) {
+            print $"- (ansi cyan)is(ansi reset) a custom command."
+        } else {
+            print $"- (ansi cyan)is not(ansi reset) a custom command."
+        }
+        if ($command.is_keyword) {
+            print $"- (ansi cyan)is(ansi reset) a keyword."
+        } else {
+            print $"- (ansi cyan)is not(ansi reset) a keyword."
         }
-    }
 
-    print ""
+        print ""
+    }
 }
 
-# Show help on nushell commands.
+# Show help on nushell commands. 
 export def "help commands" [
     ...command: string@"nu-complete list-commands"  # the name of command to get help on
     --find (-f): string  # string to find in command names and usage
+    --all (-a) # if showing help for single command, display all sections
 ] {
     let commands = ($nu.scope.commands | where not is_extern | reject is_extern | sort-by name)
 
     let command = ($command | str join " ")
-
     if not ($find | is-empty) {
         let found_commands = ($commands | find $find)
 
         if ($found_commands | length) == 1 {
-            show-command ($found_commands | get 0)
+            #todo refactor so only one invocation of show-command, because it's clunky to pass --all
+            if $all {
+                show-command --all ($found_commands | get 0)  
+            } else {
+                show-command ($found_commands | get 0)
+            }
         } else {
             $found_commands | select name category usage signatures search_terms
         }
     } else if not ($command | is-empty) {
         let found_command = ($commands | where name == $command)
-
+        
```

Comment:
I think that the check isn't failing because the PR is still a draft, but you should run `rustfmt`.